nixos/modules/services/monitoring/netdata.nix

```diff
@@ -318,6 +319,14 @@ in {
         permissions = "u+rx,g+x,o-rwx";
       };
 
+      "systemd-journal.plugin" = {
+        source = "${cfg.package}/libexec/netdata/plugins.d/systemd-journal.plugin.org";
+        capabilities = "cap_sys_admin+ep";
```

Comment:
If I read your commit message, I still cannot know why I do need `CAP_SYS_ADMIN`, is this recommended by netdata? Is this the most minimal capability? See https://man7.org/linux/man-pages/man7/capabilities.7.html for examples (`CAP_SYS_ADMIN` is explicitly a capability to **avoid**, it is too powerful).

Reply:
At least, make it clear in the code with a comment `# This is recommended by netdata in XYZ` and in the commit that the set of permissions required induce necessarily CAP_SYS_ADMIN because no other capability enable this set of needed features.

Reply:
Sure, it seems CAP_SYS_ADMIN is overkill with this plugin.
I need some time to test if limiting to:
CAP_DAC_READ_SEARCH
              •  Bypass file read permission checks and directory read
                 and execute permission checks;
is enough.
or investigate doing this with  systemd "Protect*" unit options in [systemd.exec(5)](https://man7.org/linux/man-pages/man5/systemd.exec.5.html).
any advices or help are welcome.

Reply:
Hi, capability dropped to minimal :  CAP_DAC_READ_SEARCH and CAP_SYSLOG. (see commit for details)
I have updated to 1.43.2 has 1.43.0 has some troubles.